### ISSUE TITLE
Training model with custom data

### ISSUE THREAD

**Issue by minhson-kaist** [facebookresearch/UnsupervisedDecomposition#4]
Hi, thanks for releasing the code and amazing work!

I have a question in terms of using the code.
How can I train another question decomposition model with my custom data?
In my case, I have only 1 single question per 1 multi-hop question, so the data could be a bit different.

Thanks.

**Comment by ethanjperez** [comment]
You should be able to use the same training code - Just format the multi-hop questions (one question per line) and single-hop questions (one question per line, in the same order as for multi-hop questions). If you're only trying to generate one single sub-question, you may want to alter the stopping criterion (or remove it), because the stopping criterion checks if there are two sub-questions in the decomposition (see the Appendix of our paper for details). You could also use the original XLM repo as well (which uses a small validation set for early stopping instead of round-trip BLEU in our paper). Hope that helps!
